compiler/control/OMROptions.cpp

```diff
@@ -1242,6 +1242,8 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"unleashStaticFieldFolding",        "O\tbypass the class white-list, and allow static final fields to be folded aggressively", RESET_OPTION_BIT(TR_RestrictStaticFieldFolding), "F"},
    {"unresolvedSymbolsAreNotColdAtCold", "R\tMark unresolved symbols as cold blocks at cold or lower", SET_OPTION_BIT(TR_UnresolvedAreNotColdAtCold), "F"},
    {"upgradeBootstrapAtWarm",           "R\tRecompile bootstrap AOT methods at warm instead of cold", SET_OPTION_BIT(TR_UpgradeBootstrapAtWarm), "F", NOT_IN_SUBSET},
+   {"useFlattenedFieldArrayRuntimeHelpers",        "M\tuse the runtime helpers for flattened fields for array elements", SET_OPTION_BIT(TR_UseFlattenedFieldArrayRuntimeHelpers), "F"},
+   {"useFlattenedFieldRuntimeHelpers",        "M\tuse the runtime helpers for flattened fields", SET_OPTION_BIT(TR_UseFlattenedFieldRuntimeHelpers), "F"},
```

Comment:
I think there should also be "disable" variants of these options. Maybe `dontUseFlattenedFieldRuntimeHelpers`?

Also I think `useFlattenedArrayElementRuntimeHelpers` is a more accurate name than `useFlattenedFieldArrayRuntimeHelpers`.

Reply:
I will add the options to `disable` or not use runtime helper and will update the name for array to `useFlattenedArrayElementRuntimeHelpers` in next commit.